packages/familie-header/src/søk/idnrValidator.ts

```diff
@@ -0,0 +1,60 @@
+const elevenDigits = new RegExp('^\\d{11}$')
+
+export const idnr = (digits: string, acceptSynthNr = false) => {
+   const isDnr = + digits.substr(0, 1) >= 4
+   return validate(digits, isDnr, acceptSynthNr)
+}
+
+const fixSynthNr = (digits: string) =>
+    +digits.substr(2,1) >= 8? digits.substr(0, 2) + (+digits.substr(2, 1) - 8) + digits.substr(3)
+     : +digits.substr(2,1) >= 4? digits.substr(0, 2) + (+digits.substr(2, 1) - 4) + digits.substr(3): digits
```

Comment:
I'm a big fan of shortening functions, but these are quite long lines for ternary operator, should we maybe use if else to make it more readable? 
```suggestion
const fixSynthNr = (digits: string): string => {
    if (+digits.substr(2, 1) >= 8) {
        return digits.substr(0, 2) + (+digits.substr(2, 1) - 8) + digits.substr(3);
    } else if (+digits.substr(2, 1) >= 4) {
        return digits.substr(0, 2) + (+digits.substr(2, 1) - 4) + digits.substr(3);
    }
    return digits;
};
```

Reply:
Eller noe slik
```suggestion
function subtrahereFraMåned(digits: string, number: number) {
    return digits.substr(0, 2) + (+digits.substr(2, 1) - number) + digits.substr(3);
}

const fixSynthNr = (digits: string): string => {
    if (+digits.substr(2, 1) >= 8) {
        return subtrahereFraMåned(digits, 8);
    } else if (+digits.substr(2, 1) >= 4) {
        return subtrahereFraMåned(digits, 4);
    }
    return digits;
};
```

Reply:
Cool👍